templates/contribute/tags_and_branches.md

```diff
@@ -58,23 +66,36 @@ Mathlib.
-  * Thus if `nightly-testing-YYYY-MM-DD` exists, we know that:
+  * Thus if `nightly-testing-YYYY-MM-DD` exists, we know that on it:
     * the `lean-toolchain` is `leanprover/lean4:nightly-YYYY-MM-DD`, and
     * CI succeeds.
 * When changes are required to Std to adapt to a breaking change in Lean,
-  please open a PR.
-  * Typically, if the change was made in `leanprover/lean4#NNNN`,
-    this PR should be from a branch named `lean-pr-testing-NNNN`,
-    and `lean-toolchain` will contain `leanprover/lean4-pr-releases:pr-release-NNNN`.
-  * This PR should be made against the `bump/v4.X.0` branch, not against `main`.
-  * This PR should be labelled `v4.X.0`, where `v4.X.0` is the upcoming stable version,
-    and also labelled 'depends on core changes'.
+  you will need to make a branch, and later open a PR from that branch.
+  (Note that the steps below happen automatically at Mathlib,
+  but need to be done manually for Std.)
+  * If the change was made in `leanprover/lean4#NNNN`,
+    then this branch should be called `lean-pr-testing-NNNN`.
+  * This branch should be based off the branch `nightly-testing-YYYY-MM-DD`
+    where `YYYY-MM-DD` is the date of the nightly release that your Lean PR is based off
+    (you can check this will `script/most-recent-nightly-tag.sh` in the Lean repository).
+  * If this branch does not yet exist, you will need to wait
+    (and possibly move forward to a subsequent nightly).
+    Contact @semorrison for assistance if needed.
+  * Ideally you will push this branch to the main Std repository; we can provide write access if needed.
+  * The `lean-toolchain` on this branch must contain `leanprover/lean4-pr-releases:pr-release-NNNN`.
+  * You may open a PR from the `lean-pr-testing-NNNN` branch, either before or after
+    making the required adaptations.
+  * When opening the PR, remember to set the base branch to `nightly-testing-YYYY-MM-DD`.
+  * Note that the PR, when approved, will not actually be merged into `nightly-testing-YYYY-MM-DD`
+    but rather into the `bump/v4.X.0` branch.
+  * Please label the PR with the `v4.X.0` and 'depends on core changes' labels.
+    (Or ask for this to be done if you don't have write access.)
   * Once the Lean PR has been merged and published in a nightly release, the Std adaptation PR
     * should have its `lean-toolchain` updated to `leanprover/lean4:nightly-YYYY-MM-DD`
     * may be merged into `nightly-testing` as needed.
   * Once the Std adaptation PR has been approved, it can be merged to `bump/v4.X.0`.
 * It is always allowed to merged `bump/v4.X.0` into `nightly-testing`, but not conversely.
-  (Changes to `bump/v4.X.0` have been reviewed, but changes to `nightly-testing` may not be.)
+  (Changes to `bump/v4.X.0` have been reviewed, but changes to `nightly-testing` may nothave been.)
 * When it is time to update Std to a new Lean release,
   *hopefully* all that is required is to make a new PR
   consisting of squash merging `bump/v4.X.0` to `main`.
```

Comment:
I can't comment on the title below that is not part of the diff, but it contains a typo (`leanprover-community/mathilb4`).